### PR TITLE
feat: Component Packages

### PR DESCRIPTION
**Pull request by peterinnesmsft**
### Purpose
This change is moving over a number of component packages that were previously hosted in the microsoft/botframework-virtual-assistant repo.

### Changes
Adding the following component packages to experimental/components:

- ConversationalCore
- General
- Greeting
- Onboarding
- UnknownIntent

### CODE REVIEW

File: experimental/components/ConversationalCore/assistant.dialog [hunk 1]
```diff
@@ -0,0 +1,83 @@
+{
+  "$kind": "Microsoft.AdaptiveDialog",
+  "$designer": {
+    "$designer": {
+      "name": "Assistant",
```
Comment (ryanlengel) [inline experimental/components/ConversationalCore/assistant.dialog:5]: What does it mean here that the name of this "bot" is Assistant? When the ConversationalCore is imported, do users see Assistant at all?

File: experimental/components/ConversationalCore/language-generation/en-us/common.en-us.lg [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+                    "type": "ColumnSet",
+                    "columns": [
+                        {
+                            "type": "Column",
+                            "width": 10,
+                            "items": [
+                                {
+                                    "type": "Image",
+                                    "altText": "",
+                                    "url": "https://raw.githubusercontent.com/ryanlengel/adaptivecards/master/assets/icons/e787-32.svg",
```
Comment (ryanlengel) [inline experimental/components/ConversationalCore/language-generation/en-us/common.en-us.lg:90]: This needs to be cleaned up, likely when having an official design/spec per #435

File: experimental/components/ConversationalCore/assistant.dialog [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+          "$designer": {
+            "id": "dSHrlN"
+          },
+          "activityProcessed": true,
+          "dialog": "Greeting"
+        }
+      ]
+    },
+    {
+      "$kind": "Microsoft.OnIntent",
```
Comment (ryanlengel) [inline experimental/components/ConversationalCore/assistant.dialog:29]: This is the root bot dialog, assuming that Onboarding exists. I don't think this should be in a package.

File: experimental/components/ConversationalCore/assistant.dialog [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+          "$designer": {
+            "id": "jc7tAB"
+          },
+          "activityProcessed": true,
+          "dialog": "Onboarding"
+        }
+      ]
+    },
+    {
+      "$kind": "Microsoft.OnIntent",
```
Comment (ryanlengel) [inline experimental/components/ConversationalCore/assistant.dialog:46]: Same comment as Onboarding above.

**Review by ryanlengel — CHANGES_REQUESTED** [review]

File: experimental/components/ConversationalCore/assistant.dialog (hunk 1 above)
Comment (peterinnesmsft) [reply experimental/components/ConversationalCore/assistant.dialog:5]: So, realistically, this is something that should be templatized. I think this something we'll need to figure out in conjunction with the generators and overall templating strategy. I agree that it shouldn't be 'Assistant' out of the box - but until we figure out the templating stuff, I'm not sure that I can do much right now. I'll create an issue to track this so we can make sure it gets resolved.

File: experimental/components/ConversationalCore/language-generation/en-us/common.en-us.lg (hunk 2 above)
Comment (peterinnesmsft) [reply experimental/components/ConversationalCore/language-generation/en-us/common.en-us.lg:90]: Will create a tracking issue.

File: experimental/components/ConversationalCore/assistant.dialog (hunk 1 above)
Comment (peterinnesmsft) [reply experimental/components/ConversationalCore/assistant.dialog:5]: Opened https://github.com/microsoft/botframework-components/issues/451.

File: experimental/components/ConversationalCore/assistant.dialog (hunk 4 above)
Comment (peterinnesmsft) [reply experimental/components/ConversationalCore/assistant.dialog:46]: Removed OnIntent trigger for General intent.

File: experimental/components/ConversationalCore/assistant.dialog (hunk 3 above)
Comment (peterinnesmsft) [reply experimental/components/ConversationalCore/assistant.dialog:29]: Removed OnIntent trigger for Onboarding intent.

File: experimental/components/ConversationalCore/assistant.dialog (hunk 4 above)
Comment (benbrown) [reply experimental/components/ConversationalCore/assistant.dialog:46]: How are these sub-dialogs going to be invoked then?

File: experimental/components/ConversationalCore/language-generation/en-us/common.en-us.lg (hunk 2 above)
Comment (peterinnesmsft) [reply experimental/components/ConversationalCore/language-generation/en-us/common.en-us.lg:90]: Created https://github.com/microsoft/botframework-components/issues/452.

**Review by ryanlengel — APPROVED** [review]

**Review by benbrown — APPROVED** [review]

**Pull request merged**